doc/onboarding.md

```diff
@@ -164,8 +164,14 @@ onboarding session.
 
 ## Landing PRs: Details
 
-* Please never use GitHub's green "Merge Pull Request" button.
+* Please never use GitHub's green ["Merge Pull Request"](https://help.github.com/articles/merging-a-pull-request/#merging-a-pull-request-using-the-github-web-interface) button.
   * If you do, please force-push removing the merge.
+  * Reasons for not using the web interface button:
+    * The old merge method would add an unnecessary merge commit.
```

Comment:
one more nit: it may make sense to not refer to these as old / new

more features are bound to come, and after a certain amount of time `latest` won't exactly make sense.